Content.Server/GameTicking/GameTicker.RoundFlow.cs

```diff
@@ -72,33 +72,30 @@ private void PreRoundSetup()
             _startingRound = false;
             var startTime = _gameTiming.RealTime;
             var map = _gameMapManager.GetSelectedMapChecked(true);
-            var grid = _mapLoader.LoadBlueprint(DefaultMap, map.MapPath);
+            _mapLoader.LoadMap(DefaultMap, map.MapPath);
 
-
-            if (grid == null)
+            foreach (var grid in _mapManager.GetAllMapGrids(DefaultMap))
             {
-                throw new InvalidOperationException($"No grid found for map {map.MapName}");
-            }
+                _stationSystem.InitialSetupStationGrid(grid.GridEntityId, map);
```

Comment:
This is incorrect. Only the first grid should go through setup, all grids afterwards need added to the station returned with `AddGridToStation` so that the loaded station map is treated as one.

Reply:
Are multiple stations not supported?

Seems like mappers should be the ones manually adding stationcomponent to stuff.

Reply:
> Are multiple stations not supported?
> 
> Seems like mappers should be the ones manually adding stationcomponent to stuff.

They are supported, but the GameMapPrototype infra assumes the provided file is the entire station. Doing it this way would make each shuttle/extra an identical copy of the main station down to the jobs list. (the prototype you're passing to it is used to fill the job list as well)

We can work out how to give the mapper more control, but that's how it currently works.